index.rst

```diff
@@ -31,7 +56,7 @@ A ventilation fan was installed in the AuxTel dome to aid with bringing the dome
 Second test 08-Mar-22
 =================================
 
-This test was then repeated with the dome slit open.  The plots below show the mount tracking error increasing steadily as the fan speed is increased.  The conclusion is that the mount tracking errors are caused by turbulent wind buffeting the mount.  The elevation mount tracking errors are also very similar to that seen when mount jitter is seen when observing during times of high wind speed.
+This test was then repeated with the dome slit open.  The plots below show the mount jitter increasing steadily as the fan speed is increased.  The conclusion is that the mount jitter is caused by turbulent wind buffeting the mount.  The elevation mount jitter is also very similar to that seen when observing during times of high wind speed.
```

Comment:
I'm a bit embarrassed to say that I don't really know what "wind buffeting" is. I thought it was "vibrations" from pressure changes on either side of a barrier. Is that what you mean? Or do you just mean general turbulence that shakes the telescope (e.g. wind shake)?